src/toil/provisioners/azure/azureProvisioner.py

```diff
@@ -211,8 +213,7 @@ def _checkIfClusterExists(self):
         try:
             self.callPlaybook(self.playbook['check-cluster'], ansibleArgs, wait=True)
         except RuntimeError:
-            logger.info("The cluster could not be created. Try deleting the cluster if it already exits.")
-            raise
+            raise RuntimeError("The cluster could not be created. Try deleting the cluster if it already exits.")
```

Comment:
I like this, but I wonder if there's any info from the old RuntimeError that is now lost.

Reply:
Oh from inside self.callPlaybook?